src/Compilers/VisualBasic/Portable/BoundTree/Statement.vb

```diff
@@ -1460,7 +1461,11 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 Return s_variablesMappings.GetValue(
                     Me,
                     Function(BoundUsing)
-                        Return New Variables(BoundUsing.ResourceList.As(Of IVariableDeclaration))
+                        If (BoundUsing.ResourceList.IsDefault) Then
+                            Return Nothing
+                        End If
+                        Dim usingStatementSyntax = DirectCast(Syntax.ChildNodes()(0), UsingStatementSyntax)
```

Comment:
Because I apparently misread your previous comment and it didn't occur to me :).